ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -378,15 +378,18 @@
             if(restEndPointSource.length == 0 || isPostUrlSelected){
                 Utils.showComponent(restEndPointUrlTextBox, 'div');
                 Utils.hideComponent(restEndpointConfigPath, 'div');
+                restEndPointUrlTextBox.setAttribute("data-rest-endpoint-url-validation", "");
```

Comment:
why do we need to set this attribute here if we have already set it in granite:data of the dialog?

Reply:
Yes, Barshat that is redundant, and keeping the changes only in the editDialog.js (adding the validator, removing from the granite:data) and added a cypress test for the same.